blt/src/Blt/Plugin/Commands/UsersCommands.php

```diff
@@ -66,12 +83,25 @@ public function loadUsers() {
    */
   protected function isAdminBlockValid($admin) {
     // @todo This should be more robust and check bad chars.
-    return (
-      array_key_exists('username', $admin) &&
-      $admin['username'] != '' &&
-      array_key_exists('password', $admin) &&
-      $admin['password'] != ''
-    );
+    // Validate username and password.
+    $userAndPasswordValid = FALSE;
+    if (array_key_exists('username', $admin) &&
+        $admin['username'] != '' &&
+        array_key_exists('password', $admin) &&
+        $admin['password'] != '') {
+      $userAndPasswordValid = TRUE;
+    }
+    // If present, the enabled key must be boolean.
+    // It is also valid if the key is absent.
+    $enabledSectionValid = FALSE;
+    if (array_key_exists('enabled', $admin) && is_bool($admin['enabled'])) {
```

Comment:
I'm not quite sure what's going on and why `is_bool` is not operating as I expect it to, because this seems valid, but this block is passing the admin check:

```yaml
admin:
  username: 'admin'
  password: 'password123'
  enabled: 'this is not a valid boolean'
```

Reply:
I have tested this scenario and is_bool is returning false. Hence it is blocking the admin user.